File: src/main/java/com/core/base/corebase/repository/ReviewRepository.java
package com.core.base.corebase.repository;

import com.core.base.corebase.domain.review.Review;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface ReviewRepository extends JpaRepository<Review, Long> {
    // Add custom query methods here if needed
}

File: src/main/kotlin/com/core/base/corebase/repository/ReviewRepository.kt
package com.core.base.corebase.repository

import com.core.base.corebase.domain.review.Review
import org.springframework.data.mongodb.repository.MongoRepository
import org.springframework.stereotype.Repository
import java.util.*

@Repository
interface ReviewRepository : MongoRepository<Review, Long> {
    fun findById(id: UUID): Optional<Review>
    fun findByReviewerIdsIn(reviewerId: UUID): List<Review>
}